minetest/minetest
# Enable shadows by default in devtest

This is to address a problem raised by @NathanSalapat in a [comment](https://github.com/minetest/minetest/pull/11944#issuecomment-1081323325) to #11944.

What's done;
* Shadow is set to 0.33 for each player that joins
* /set_lighting command removed
* /set_shadow command added to control shadow parameters (intensity for now)
* All code is moved to `util_commands` mod

## To do

This PR is Ready for Review.

## How to test

* Enable shadows in your client
* Start devtest
* Game should render with dynamic shadows
